doc/source/f2py/f2py.getting-started.rst

```diff
@@ -17,7 +17,8 @@ following steps:
     functions, which can make them "smarter" and more "Pythonic".
 
 * F2PY reads a signature file and writes a Python C/API module containing
-  Fortran/C/Python bindings.
+  Fortran/C/Python bindings.  For this you need the ``python3-dev``
+  package in addition to your standard Python installation.
```

Comment:
The note makes more sense in the next bullet point as that is where the compilation happens.

I suspect `python3-dev` is likely only a helpful pointer on linux distros.  Maybe linking to generic python docs ("extension module") or add that it compiles against NumPy (we should have some basic docs for that).